app/renderer/components/tabs/tab.js

```diff
@@ -323,7 +336,10 @@ class Tab extends React.Component {
           ref={(node) => { this.tabSentinel = node }}
           className={css(styles.tab__sentinel)}
         />
-        <div className={css(styles.tabId)}>
+        <div className={css(
+          styles.tab__identity,
+          this.props.centralizeTabIcons && styles.tab__content_centered
```

Comment:
Shouldn't this be `tab__identify_centered` ?

Reply:
all things that need to be centered are now relying on `tab__content_centered` not only the `tab__identity`. not sure what to comment but lmk what you want to and I'll add

Reply:
I think this conversation itself should serve as a comment for who are confused ;-) Let's leave it as it is.